ds_judgements_public_ui/templates/pages/publishing_policy.html

```diff
@@ -70,7 +42,7 @@ <h2 id="section-why">Why we publish judgments and decisions</h2>
             it in the Digital Records Infrastructure. It creates digital surrogates of the record for publication and reuse, which are provided in Find Case Law.
           </p>
           <h2 id="section-we-publish">How we publish judgments and decisions</h2>
-          <h3 id="section-we-receive">How we receive judgments and decisions</h3>
```

Comment:
I'd be tempted to leave the ID tags in -- people might have bookmarked these links.

Reply:
@dragon-dxw updated to leave them in